neuro_san/test/driver/assert_capture.py

```diff
@@ -11,7 +11,7 @@
 from typing import Any
 from typing import List
 
-from tests.framework.interfaces.assert_forwarder import AssertForwarder
+from neuro_san.test.interfaces.assert_forwarder import AssertForwarder
```

Comment:
Move of test infrastructure to be shipping code implies that a whole bunch of imports will change.